src/plugins/kibana_usage_collection/server/collectors/ui_metric/telemetry_ui_metric_collector.ts

```diff
@@ -66,9 +66,9 @@ export function registerUiMetricUsageCollector(
           attributes: { count },
         } = rawUiMetric;
 
-        const [appName, ...metricType] = id.split(':');
+        const [appName, metricType] = id.split(':');
 
-        const pair = { key: metricType.join(':'), value: count };
```

Comment:
It is reverting a fix we just merged this morning (https://github.com/elastic/kibana/pull/76913). 

Maybe an issue when updating the branch from `master`?

Reply:
I did not touch this file